trestle/tasks/xlsx_to_oscal_profile.py

```diff
@@ -123,20 +126,90 @@ def _execute(self) -> TaskOutcome:
         self._report_issues()
         return TaskOutcome('success')
 
-    def _get_include_controls(self) -> List[SelectControlById]:
-        """Get include controls from spread sheet."""
-        include_control = SelectControlById(with_ids=self._get_with_ids())
-        return [include_control]
+    def _get_imports_by_goal(self) -> List[Import]:
+        """Get imports by goal."""
+        return self._get_imports_by_check()
```

Comment:
Do you really need this other entry for get_imports_by_check()?  Maybe it makes the code more clear.

Reply:
I think this is fine for now. It is a way to document that `by_goal` and `by_check` mean the same thing.